plan/logical_plan_builder.go

```diff
@@ -1215,6 +1206,14 @@ func (b *planBuilder) buildDataSource(tn *ast.TableName) LogicalPlan {
 		return nil
 	}
 	tableInfo := tbl.Meta()
+	handle := sessionctx.GetDomain(b.ctx).StatsHandle()
+	var statisticTable *statistics.Table
+	if handle == nil {
```

Comment:
Any test case covers this?

Reply:
This is a move-equivalent action.
PTAL @lamxTyler

Reply:
It is move-equivalent.